integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/commands/AbstractRequestCmd.java

```diff
@@ -440,4 +439,14 @@ private boolean isUpdate() {
     private boolean isCreateOrUpdate() {
         return "post".equals(httpVerb) || "put".equals(httpVerb);
     }
+
+    private boolean isGetByID(String url) {
+        String[]  unGetByIDUrls = {"clients-initial-access"};
```

Comment:
This may be even clearer if you just did 

return !Arrays.asList("clients-initial-access").contains(url);

Or, if there are no other known urls of this type for now, just do:

return !"clients-initial-access".equals(url);

And let the next person decide how to account for multiple urls.

Reply:
This looks great. I'm sorry for not seeing the reply in time due to some things.